kclvm/runner/src/assembler.rs

```diff
@@ -37,8 +36,8 @@ pub(crate) trait LibAssembler {
     /// e.g. LLVM IR -> return : ".ll"
```

Comment:
Change .ll to .o.
Do a global search on it.

Reply:
> Change .ll to .o. Do a global search on it.

fixed.